README.md

```diff
@@ -1,725 +1,13 @@
-indicate different states or checkpoints with the `:state-start:` and
-`:state-end:` commands:
-
-```swift
-// WelcomeViewController.swift
-
-// ... more code ...
-// :snippet-start: sign-up
-@objc func signUp() {
-    // :state-start: final
-    setLoading(true);
-    app.emailPasswordAuth.registerUser(email: email!, password: password!, completion: { [weak self](error) in
-        DispatchQueue.main.async {
-            self!.setLoading(false);
-            ...
-        }
-    })
-    // :state-end:
-    // :state-start: start
-    // TODO: Use the app's emailPasswordAuth to registerUser with the email and password.
-    // When registered, call signIn().
-    // :state-uncomment-end:
-}
-// :snippet-end:
-// ... more code ...
-```
-
-Running `bluehawk copy` on this file with `--state start` results in a copy of
-`WelcomeViewController.swift` that looks something like this:
-
-```swift
-// WelcomeViewController.swift
-
-// ... more code ...
-@objc func signUp() {
-    // TODO: Use the app's emailPasswordAuth to registerUser with the email and password.
-    // When registered, call signIn().
-}
-// ... more code ...
-```
-
-Notice that you still have all of the boilerplate, but no final implementation
-code. Only the "TODO" is left.
-
-Using the `--state final` flag produces another version of
-`WelcomeViewController.swift` that has the boilerplate and the final
-implementation code, but no "TODO":
-
-```swift
-// WelcomeViewController.swift
-
-// ... more code ...
-@objc func signUp() {
-    setLoading(true);
-    app.emailPasswordAuth.registerUser(email: email!, password: password!, completion: { [weak self](error) in
-        DispatchQueue.main.async {
-            self!.setLoading(false);
-            ...
-        }
-    })
-}
-// ... more code ...
-```
-
-You can run Bluehawk on an entire directory, and each file in the repo will be
-copied or transformed to the destination. This makes it easy to copy one state
-of the entire tutorial source into another repo that learners can clone.
-
-## Plugins
-
-You can add commands and listeners by creating a JS file or node project that
-implements the register() function:
-
-```js
-// myPlugin.js
-exports.register = (bluehawk) => {
-  // Register a new command, :my-command:
-  bluehawk.registerCommand("my-command", {
-    rules: [],
-    process: (request) => {
-      // Execute command
-    },
-  });
-
-  // Register a document listener
-  bluehawk.subscribe((finishedDocument) => {
-    // Do something with finishedDocument
-  });
-};
-```
-
-Usage:
-
-```shell
-bluehawk --plugin ./myPlugin source.txt
-```
-
-You can pass the --plugin flag multiple times to load different plugins or create a plugin that is composed of other plugins.
-
-## Usage as a Module
-
-```sh
-npm install bluehawk
-```
-
-## Contributing
-
-For more information about how to run, build, or test Bluehawk yourself, see [CONTRIBUTING.md](CONTRIBUTING.md).
```

Comment:
Why is README completely deleted?

Reply:
Even if we remove the actual docs/reference, I think we could keep the Example, Videos, and Install sections.

Reply:
absolutely no idea. good catch. will add back.

Reply:
update on this. if you expand the readme (i think [this link](https://github.com/mongodb-university/Bluehawk/pull/74/files/823f03266681f3691f8618901e0919b396e1afd4) will work), you should see what's left of it. it was just dramatically reduced in size b/c i moved most of the content from this 1 file to various files in the docusaurus site.